### PR TITLE
feat: Slashing and MEV stealing guides for CSM

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

**Review by skhomuti — APPROVED** [review]

File: docs/staking-modules/csm/guides/slashing.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+:::info
+Despite the common misconception, being offline is not a slashable event.
+:::
+
+## What are the slashing penalties?
+
+There are three penalties associated with the slashing:
+
+- **Initial penalty** also known as minimal penalty. Applied at the moment of slashing reporting to CSM. Amount = `effective_balance / 32` (usually 1 ETH)
```
Comment (madlabman) [inline docs/staking-modules/csm/guides/slashing.md:26]:
```suggestion
- **Initial penalty** also known as minimal penalty. Applied at the moment of slashing reporting to CSM. Amount = `effective_balance // 32` (usually 1 ETH)
```

File: docs/staking-modules/csm/guides/slashing.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- **Midterm penalty** also known as correlation penalty. Applied in the middle of the slashing period (day #18). The amount may vary from `0 ETH to 32 ETH` depending on the number of other ongoing slashings (usually `0 ETH)`
```
Comment (madlabman) [inline docs/staking-modules/csm/guides/slashing.md:27]:
```suggestion
- **Midterm penalty** also known as correlation penalty. Applied in the middle of the slashing period (day #18). The amount may vary from `0 ETH to 32 ETH` depending on the number of other ongoing slashings (usually `0 ETH`)
```

File: docs/staking-modules/csm/guides/slashing.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- **Misses attestation penalty**. During the whole period of slashing (usually 36 days), the validator balance is penalized for the missed attestations as if the validator was offline (usually `~0.1-0.2 ETH`)
```
Comment (madlabman) [inline docs/staking-modules/csm/guides/slashing.md:28]:
```suggestion
- **Missed attestations penalty**. During the whole period of slashing (usually 36 days), the validator balance is penalized for the missed attestations as if the validator was offline (usually `~0.1-0.2 ETH`)
```

File: docs/staking-modules/csm/guides/mev-stealing.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- **Do nothing.** In this case, a penalty would be proposed to the Lido DAO for confirmation by the EasyTrack motion. If the LDO holders do not object to the motion, the MEV detection committee will enact it, and bond funds will be burned. If, after the burn, the bond is not sufficient to cover all uploaded keys, either new deposits to the [unbonded keys](../../csm/join-csm#unbonded-validators) will be suspended, or, if all of the keys are already deposited, unbonded validators will be requested to exit. You can read more on the measures applied in case of not exiting the validators [here](../../csm/validator-exits#protocol-initiated-exits). Also, upon penalty confirmation, the bond curve will be [reset](../../csm/penalties.md#benefits-reset) for the Node Operator if they previously had a beneficial bond curve.
```
[the remark this answers is not in the document]
Comment (dgusakov) [reply docs/staking-modules/csm/guides/mev-stealing.md]: Resolved manually

**Review by madlabman — APPROVED** [review]

**Pull request merged**